src/components/Modal/ModalHeader/ModalHeader.tsx

```diff
@@ -10,9 +10,9 @@ import styles from "./ModalHeader.module.scss";
 
 export interface ModalHeaderProps extends VibeComponentProps {
   /**
-   * Title of the modal
+   * Title of the modal - using string is a recommended standard
```

Comment:
worth explaining why it's recommended.
It's recommended because it will provide the well-defined styles of a title.
Using it by passing other content should be according to design guidlines

Reply:
Expanded the description here